recipes/scikit-sparse/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.3" %}
+
+package:
+  name: scikit-sparse
+  version: {{ version }}
+
+source:
+  fn: scikit-sparse--{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/scikit-sparse/scikit-sparse-{{ version }}.tar.gz
+  sha256: 2456be16e65bb8126ea7b9a3b175282b396d07ebeb0b4c5ac85d707bfc977008
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - numpy x.x
+    - suitesparse
+    - cython
+    - setuptools
+  run:
+    - python
+    - numpy x.x
+    - scipy
+    - suitesparse
+    - setuptools
```

Comment:
I don't think it should be, but the tests wouldn't pass without it.  Looking into it, there is one place in the tests that does:
`from pkg_resources import resource_filename`
I was getting an ImportError on pkg_resources.  I will try moving it under test/requires instead and see how that goes.